docker/compose/build/install-configs.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+ecosystem_name=$1
+agent_host_address=$2
+if [ -z "$ecosystem_name" ] || [ -z "$agent_host_address" ]; then
+    echo "Usage: ./install-configs.sh <ecosystem> <agent host address>"
+    exit 1
+fi
+
+# Check if agent_host_address contains 'http://' or 'https://'
+if [[ ! $agent_host_address =~ ^http ]]; then
+    echo "agent host address should be an http/https URL"
+    exit 1
+fi
+
+# copy & patch .env files from templates
+mkdir -p ./agent
+cp ../../../packages/agent/.env.example ./agent/.env.local
+sed -i "s|^CONF_PATH=.*|CONF_PATH=\"/opt/oid4vc-demo/packages/agent/conf/${ecosystem_name}\"|" ./agent/.env.local
```

Comment:
please add another check here for the `ecosystem_name` the config for the name doesn't exist, we should exit

Reply:
fixed